bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitOHItemProxy.java

```diff
@@ -48,18 +47,24 @@
     public static final String BRIGHTNESS_COMMAND = "brightness";
     public static final String ON_COMMAND = "on";
 
-    // delay, how long wait for further commands.
-    // TODO: make it configurable ?
-    private final int DELAY = 50;
+    public static final int DIMMER_MODE_NONE = 0;
```

Comment:
```suggestion
    public static final int DIMMER_MODE_NORMAL = 0;
```

I would suggest an enum for the mode: NORMAL, SUPPRESS_ON, SUPPRESS_BRIGHTNESS, SMART with explantation as comments

Reply:
good idea. done

Reply:
you could also do this for the command types above